src/content/docs/guides/dingo/SPO Guides/005-register-pool.md

```
@@ -0,0 +1,311 @@
+## Step 9 - Update your `dingo.yaml` with the new KES key, VRF key and operation certificate
+
+⚠️ On Block Producer
+
+Stop the Dingo node by running:
+```
+sudo systemctl stop dingo
+```
+
+Add the following lines to your `dingo.yaml` file by running:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)